craft_providers/bases/buildd.py

```diff
@@ -827,12 +814,9 @@ def _setup_wait_for_network(
         logger.debug("Waiting for networking to be ready...")
 
         _check_deadline(deadline)
+        command = ["getent", "hosts", "snapcraft.io"]
```

Comment:
pedantic: It would be nice if this network check used the same bash tcp call as the new one